bedrock/firefox/templates/firefox/includes/release-menu.html

```diff
@@ -0,0 +1,11 @@
+<nav id="nav-main" role="navigation">
+  <span id="nav-main-toggle" class="toggle" role="button" aria-controls="nav-main-menu" aria-expanded="false" tabindex="0">{{_('Menu')}}</span>
+  <ul id="nav-main-menu">
+    <li class="first"><a class="more" rel="external" href="https://developer.mozilla.org">{{_('MDN')}}</a>
```

Comment:
> Is there a reason why only the MDN link has class="more"?

This was also before my time. I presume this is to help indicate that the MDN link is "external". The semantics of the class name aren't great, but that isn't really user-visible.